modules/apps/object/object-test/src/testFunctional/paths/ObjectAction.path

```diff
@@ -72,6 +72,11 @@
 		<td>//h3[(@class='lfr-objects__card-title') and text()='${key_title}']</td>
 		<td></td>
 	</tr>
+	<tr>
+		<td>ERROR_MESSAGE_STANDALONE_ACTION</td>
+		<td>//div//div[contains(@class,'form-group')]//label[text()='Message']/..//input[@type='text']</td>
+		<td></td>
+	</tr>
```

Comment:
I guess that there is a path to this. Look on Custom Validation, because it's the same structure as Error Message on Validation.

Maybe: `ADD_ERROR_MESSAGE_VALIDATION`